services/blockchain-indexer/shared/dataService/business/invoke.js

```diff
@@ -67,12 +70,33 @@ const validateEndpointParams = async invokeEndpointParams => {
 	}
 };
 
+const checkIfEndpointAllowed = endpoint => {
+	if (INVOKE_ALLOWED_METHODS.includes('*')) {
+		return true;
+	}
+
+	// eslint-disable-next-line no-restricted-syntax
+	for (const allowedMethod of INVOKE_ALLOWED_METHODS) {
+		if (endpoint.includes(allowedMethod)) {
```

Comment:
This loop can also be replaced with `Array.some` and response can be directly returned.
```suggestion
		if (endpoint.startsWith(allowedMethod)) {
```